addons/hr_holidays/static/src/js/time_off_calendar.js

```diff
@@ -9,6 +9,7 @@ odoo.define('hr_holidays.dashboard.view_custo', function(require) {
     var CalendarModel = require("web.CalendarModel");
     var CalendarView = require("web.CalendarView");
     var viewRegistry = require('web.view_registry');
+    var FullCalendar = require('/web/static/src/js/libs/fullcalendar.js');
```

Comment:
You can remove this import as it's not used :-)

Reply:
Done, thanks! :)